content/reference/dos-mitigation.md

```diff
@@ -3,6 +3,8 @@ title: "DOS Mitigation"
 weight: 3
 ---
 
+><em> This document is up to date as of go-libp2p [v0.22.0](https://github.com/libp2p/go-libp2p/releases/tag/v0.22.0)</em>
```

Comment:
Is the expectation that this doc will be updated on every release? That seems untenable and not a great use of time.

Reply:
I don't think this doc (and website) should follow the same release process as the codebase and agree that it's unreasonable to update this every release. Since we make API references in places, this note was to let the reader know that it's using v0.22
I can remove it if you think it's confusing.

Reply:
I removed it